tools/bazar/actions/BazarListeAction.php

```diff
@@ -118,6 +130,10 @@ public function formatArguments($arg)
             // paramètre de tri des fiches sur une date (en gardant la retrocompatibilité avec le paramètre agenda)
             'agenda' => $arg['datefilter'] ?? $arg['agenda'] ?? null,
             'datefilter' => $arg['datefilter'] ?? $arg['agenda'] ?? null,
+            // Dynamic mean the template will be rendered from the front end in order to improve UX and perf
+            // Only few bazar templates have been converted to javascript
+            'dynamic' => $dynamic,
```

Comment:
```suggestion
            'dynamic' => $dynamic && !$externalModeActivated,
```
Je serai d'avis de ne pas activer le mode dynamique pour les données externes (pour le moment)